File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AncientGloves.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class AncientGloves extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_ancient_gloves_name;
        this.idDescription = R.string.accessory_ancient_gloves_description;
        this.idImage = R.drawable.ancient_gloves;
        this.price = 468L;
        this.constitution = 27;
        this.dexterity = 22;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/AncientGloves.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class AncientGloves : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_ancient_gloves_name
        idDescription = R.string.accessory_ancient_gloves_description
        idImage = R.drawable.ancient_gloves
        price = 468L
        constitution = 27
        dexterity = 22
    }
}
